dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -4,32 +4,31 @@
     <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
   </div>
   <div v-if="mapOfKpiKeysToDataObjects.size > 0 && !waitingForData">
-    <LksgCompanyDataTable
+    <P2pCompanyDataTable
```

Comment:
LksgPanel.vue but P2pCompanyDataTable component. This must be checked again.

Reply:
Naming is corrected to DisplayFrameworkDataTable.
Also corrected it in p2p panel.